pkg/gatherers/clusterconfig/ceph_cluster.go

```diff
@@ -5,24 +5,40 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/openshift/insights-operator/pkg/record"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
 )
 
-// GatherCephCluster collects statuses of the`cephclusters.ceph.rook.io` resources
+// GatherCephCluster Collects statuses of the`cephclusters.ceph.rook.io` resources
 // from Openshift Data Foundation Stack.
 //
-// API Reference:
+// ### API Reference:
+// https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+//
+// ### Sample data:
+// docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
+//
+// ### Location in archive:
+// | Version | Path													 |
+// | ------- | ----------------------------------------------------- |
+// | <= 4.12 | config/storage/{namespace}/{name}.json 				 |
+// | 4.13+   | config/storage/{namespace}/cephclusters/{name}.json 	 |
+//
+// ### Config ID:
+// clusterconfig/ceph_cluster
 //
-//	https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+// ### Released version:
```

Comment:
Other gatherers are using a single list "Since versions" instead of "Release version" and "Backport versions". Is this a new norm? If so:

* Do you plan to update documentation of all gatherers? 
* Could you always include both sections, possibly saying "None" in "Backport versions" to prevent any doubts?

Reply:
I would keep `Since versions`

Reply:
Keeping only the `Since versions` is confusing. I would prefer to know what version this was released in and the versions that it was backported. It is clear, with no space for interpretation.

- Yes. I plan to update all the documentation. Separated PR soon. (we need to validate it, after all - I found a lot of inconsistencies)
- Yes. I can add `None` for `Backport versions`.

Reply:
I don't see how we could use that distinction. We care about what OCP versions have the data, not how the change got to those versions.

Anyway, feel free to go ahead with the change if it helps you somehow.